src/js/letters/utils/helpers.jsx

```diff
@@ -244,3 +245,30 @@ export const benefitOptionsMap = {
   serviceConnectedPercentage: 'serviceConnectedEvaluation',
   militaryService: 'militaryService'
 };
+
+export function isDomesticAddress(address) {
+  return (address.type === 'DOMESTIC');
+}
+
+export function isInternationalAddress(address) {
+  return (address.type === 'INTERNATIONAL');
+}
+
+export function isMilitaryAddress(address) {
+  return (address.type === 'MILITARY');
+}
+
+export function getZipCode(address) {
+  if (isInternationalAddress(address)) {
+    return '';
+  }
+  const parts = [
+    address.zipCode,
+    address.zipSuffix ? `-${address.zipSuffix}` : ''
+  ];
+  return parts.join('');
+}
+
+export function getStateName(stateCode) {
+  return STATE_CODE_TO_NAME[stateCode] || '';
```

Comment:
Should we throw something to sentry if we're trying to get a state name that isn't in our list? That might be better in another PR, but...just a thought. :man_shrugging:

Reply:
👍  SGTM either now or eventually, but maybe now is best